test/extended/fixtures/forcepull-false-s2i-is.json

```diff
@@ -0,0 +1,14 @@
+{
+    
+    "kind": "ImageStream",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "s2i-forcepull-false-ruby-20-centos7",
+        "creationTimestamp": null
+    },
+    "spec": {},
+    "status": {
+        "dockerImageRepository": ""
+    }
+    
+}
```

Comment:
arguably you could also combine both of those things into the existing BC fixtures too (so have a single fixture that defines the "setup" BC, "setup" IS, and the "test" BC all in one file) but there might be a reasonable argument to keep those 2 things (setup vs test) separate.